x-pack/solutions/security/packages/kbn-scout-security/src/playwright/fixtures/test/page_objects/security_navigation.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Locator, ScoutPage } from '@kbn/scout';
+
+export class SecurityNavigation {
+  public dashboardsButton: Locator;
+  public dashboardsLink: Locator;
+  public overviewLink: Locator;
+  public detectionResponseLink: Locator;
+  public entityAnalyticsLink: Locator;
+  public alertsLink: Locator;
+  public timelinesLink: Locator;
+  public exploreButton: Locator;
+  public exploreLink: Locator;
+  public hostsLink: Locator;
+  public networkLink: Locator;
+  public usersLink: Locator;
+  public casesLink: Locator;
+
+  constructor(private readonly page: ScoutPage) {
+    this.dashboardsButton = this.page.testSubj.locator('solutionSideNavItemButton-dashboards');
+    this.dashboardsLink = this.page.testSubj.locator('solutionSideNavItemLink-dashboards');
+    this.overviewLink = this.page.testSubj.locator('solutionSideNavPanelLink-overview');
+    this.detectionResponseLink = this.page.testSubj.locator(
+      'solutionSideNavPanelLink-detection_response'
+    );
+    this.entityAnalyticsLink = this.page.testSubj.locator(
+      'solutionSideNavPanelLink-entity_analytics'
+    );
+    this.alertsLink = this.page.testSubj.locator('solutionSideNavItemLink-alerts');
+    this.timelinesLink = this.page.testSubj.locator('solutionSideNavItemLink-timelines');
+    this.exploreButton = this.page.testSubj.locator('solutionSideNavItemButton-explore');
+    this.exploreLink = this.page.testSubj.locator('solutionSideNavItemLink-explore');
+    this.hostsLink = this.page.testSubj.locator('solutionSideNavPanelLink-hosts');
+    this.networkLink = this.page.testSubj.locator('solutionSideNavPanelLink-network');
+    this.usersLink = this.page.testSubj.locator('solutionSideNavPanelLink-users');
+    this.casesLink = this.page.testSubj.locator('solutionSideNavItemLink-cases');
+  }
+
+  async goToTimelines() {
```

Comment:
Following Playwright's `page.goto()` casing, we've named Scout's navigation functions using the same convention. For example:

```suggestion
  async gotoTimelines() {
```